ecommers/lib/core/services/membership_service.dart

```diff
@@ -17,50 +18,45 @@ class MembershipService {
   bool get isNotExpired =>
       _expirationDate != null && DateTime.now().isBefore(_expirationDate);
 
-  MembershipService(this.secureStorage);
+  MembershipService();
+
+  Future initialize() async {
+    _storage = await SharedPreferences.getInstance();
+    load();
```

Comment:
I don't understand what load

Reply:
Anyway, if I rename it to loadData, you will find that method and try to understand what the data is

Reply:
loadDataFromStorage

Reply:
loadSessionData